gradle/publishing.gradle

```diff
@@ -62,8 +63,8 @@ publishing {
         }
     }
     def publishBuild = Boolean.parseBoolean(findProperty('publishBuild') ?: 'false')
+    def isReleaseBuild = Boolean.parseBoolean(findProperty('releaseBuild') ?: 'false')
```

Comment:
did we need to pull this out?

Reply:
No, I guess not. I think I'd done that during part of the refactoring for this, but I don't think it's actually needed (at least not yet). At some point, we may want to so that we could use a different `repoKey` for snapshot and release builds, but not at the moment

Reply:
ok well its not a big deal we can keep it here.

Reply:
Reverted in c1822fc